File: Basic/shift left/shift_left.py

array=[12,3,4,5,6,6]


def shift_left(array, times):
	while times !=0:
		temp=array[0]
		array_2=['empty']*len(array)
		for i in range(1, len(array)):
			array_2[i-1]=array[i]
		array_2[len(array_2)-1]=temp
		array=array_2
		times+=-1
	print(array_2)

shift_left(array, 6)

File: Basic/shift left/shift_left.rb

array=[1,2,3,4,5,6]

def shift_left(array, times)
	while times !=0
		array_2=[]
		temp=array[0]
		(1..array.length).each do |i|
			array_2[i-1]=array[i]
		end
		array_2[array_2.length-1]=temp
		array=array_2
		times+=-1
	end
	puts array
end

shift_left(array,3)





 